src/classes/AbstractGenerator.js

```diff
@@ -328,10 +328,10 @@ class AbstractGenerator {
    * @returns {string}
    */
   fixMarkdownLinks(mdBody, context) {
-    // prefix links with path to root
+    // prefix links with path to root + add trailing "/"
     mdBody = mdBody.replace(
-      /([^!]\[[^\]]*])\(\/([a-z0-9/#-]*)\)/g,
-      `$1(${context.pathToRoot}/$2)`
+      /([^!]\[[^\]]*])\(\/([a-z0-9-]+)(?:\/(#[a-z0-9-]+)?)?\)/g,
```

Comment:
just to be 100%sure, this also handles anchors ?

Reply:
yes it does